care/users/migrations/0024_passwordresettoken.py

```
@@ -0,0 +1,32 @@
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0023_user_prefix_user_suffix'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='PasswordResetToken',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)